doc/modules/h_statistic.rst

```diff
@@ -0,0 +1,130 @@
+===============================================================
+
+.. currentmodule:: sklearn.inspection
+
+What is the difference between a white box model and a black box model?
+It is the many and complicated interaction effects of the latter.
+
+Such interaction effects can be visualized by two-dimensional or stratified
+partial dependence plots (PDP). But how to figure out between *which feature pairs*
+the strongest interactions occur?
+
+One approach is to study pairwise H-statistics, introduced by Friedman and Popescu
+in [F2008]_. The H-statistic of two features provides the proportion of effect
+variability of the two features coming from their pairwise interaction.
+
+The figure below shows H-statistics and their unnormalized counterparts for
+the bike sharing dataset, with a
+:class:`~sklearn.ensemble.HistGradientBoostingRegressor`:
+
+.. figure:: ../auto_examples/inspection/images/sphx_glr_plot_partial_dependence_010.png
+   :target: ../auto_examples/inspection/plot_partial_dependence.html
+   :align: center
+   :scale: 70
+
+The statistics have been compured for the five most important features.
+
+Mathematical definition
+=======================
+
+**Partial dependence**
+
+Let :math:`F: \mathbb{R}^p \to \mathbb{R}` denote the prediction function that 
+maps the :math:`p`-dimensional feature vector :math:`\mathbf x = (x_1, \dots, x_p)`
+to its prediction.
+Furthermore, let :math:`F_s(\mathbf x_s) = E_{\mathbf x_{\setminus s}}(F(\mathbf x_s, \mathbf x_{\setminus s}))`
+be the partial dependence function of :math:`F` on the feature subset 
+:math:`\mathbf x_s`, where :math:`s \subseteq \{1, \dots, p\}`, 
+Here, the expectation runs over the joint marginal distribution of features 
+:math:`\mathbf x_{\setminus s}` not in :math:`\mathbf x_s`.
+
+Given data, :math:`F_s(\mathbf x_s)` can be estimated by the empirical partial 
+dependence function
+
+.. math::
+  \hat F_s(\mathbf x_s) = \frac{1}{n} \sum_{i = 1}^n F(\mathbf x_s, \mathbf x_{i \setminus s}),
+
+where :math:`\mathbf x_{i\setminus s}`, :math:`i = 1, \dots, n`, 
+are the observed values of :math:`\mathbf x_{\setminus s}` in some "background" dataset.
+
+**Pairwise H-statistic**
+
+Following [F2008]_, if there are no interaction effects between features
+:math:`x_j` and :math:`x_k`, their two-dimensional partial dependence function
+:math:`F_{jk}` can be written as the sum of the univariate partial dependencies, i.e.,
+
+.. math::
+  F_{jk}(x_j, x_k) = F_j(x_j) + F_k(x_k).
+
+Correspondingly, Friedman and Popescu's H-statistic of pairwise interaction strength
+is defined as
+
+.. math::
+
+  H_{jk}^2 = A_{jk} / B_{jk},
+
+where
+
+.. math::
+
+  A_{jk} = \frac{1}{n} \sum_{i = 1}^n\big[\hat F_{jk}(x_{ij}, x_{ik}) - \hat F_j(x_{ij}) - \hat F_k(x_{ik})\big]^2
+
+and
+
+.. math::
+
+  B_{jk} = \frac{1}{n} \sum_{i = 1}^n\big[\hat F_{jk}(x_{ij}, x_{ik})\big]^2.
+
+Remarks
+=======
+
+1. Partial dependence functions and :math:`F` are centered to mean 0.
```

Comment:
The prediction function F isn’t centered, is it?

Reply:
You are right! Centered F would only necessary for univariate H-statistics